build.xml

```diff
@@ -60,6 +60,7 @@
 			<arg value="install"/>
 			<arg value="--working-dir"/>
 			<arg path="build-cs"/>
+			<arg value="--ignore-platform-reqs"/>
```

Comment:
Seems the only solution so far https://github.com/mhujer/yaml-sort-checker/pull/14/files#diff-6ac3f79fc25d95cd1e3d51da53a4b21b939437392578a35ae8cd6d5366ca5485